project/app/Helpers/FilterItemHelper.php

```diff
@@ -16,10 +16,23 @@ trait FilterItemHelper
 
     public function filterProcedure(Request $request, string $type): Response
     {
+        $user_role = $this->getRole($this->ensureIsNotNullUser(Auth::user())->role);
+
         $collection = $this->filterHelper($request, $type);
 
+        $add_form = 0;
+        if ($user_role->storage & 8) {
```

Comment:
Zmieniłem uprawnienia pracownika u admina (dodałem mu możliwość zmiany ilości przedmiotów) i w sumie nie pokazują się te rzeczy do zwiększenia ilości, nie wiem gdzie dokładnie może być błąd.

Reply:
Sprawdzę to

Reply:
Myślę, że tym może Adam się też zająć, jak wleci jego pull-request i popatrzeć co tam można pozmieniać

Reply:
Zajmę się tym, jak wlecą oba pull-requesty, to będzie łatwiej mi poprzeglądać, co tam się dzieje